app/src/main/java/io/github/droidkaigi/confsched/util/DateUtil.java

```diff
@@ -47,13 +40,15 @@ public static String getHourMinute(Date date, Context context) {
 
     @NonNull
     public static String getLongFormatDate(Date date, Context context) {
-        Locale locale = AppUtil.getCurrentLocale(context);
         if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR2) {
-            String pattern = DateFormat.getBestDateTimePattern(locale, FORMAT_YYYYMMDDKKMM);
-            return new SimpleDateFormat(pattern, locale).format(date);
+            String pattern = DateFormat.getBestDateTimePattern(Locale.getDefault(), FORMAT_YYYYMMDDKKMM);
+            return new SimpleDateFormat(pattern).format(date);
         } else {
-            SimpleDateFormat sdf = (SimpleDateFormat) java.text.DateFormat.getDateInstance(java.text.DateFormat.LONG, locale);
-            return sdf.format(date) + getHourMinute(date, context);
+            java.text.DateFormat dayOfWeekFormat = SimpleDateFormat.getDateInstance(SimpleDateFormat.LONG);
```

Comment:
Why do you use SimpleDateFormat? getDateInstance is taken over from DateFormat right? :laughing: